### DISCUSSION TITLE
about result

### DISCUSSION THREAD

**Discussion by ningqing123** [ethvis/qd-track#23] [Q&A] [answered]
I tested the code on the BDD tracking verification set, and some results show that the MOTA is higher than that of the second edition papers, and some are lower, such as Pedestrian is 46.7(48),Motocycle is 33.5( 22.4). Did I do something wrong?

**Comment by OceanPang** [comment, marked as answer]
Hi @ningqing123, the model released is not the exact one as the one reported in the paper. 
In this case, the results of each category may be different, however, the overall performance should be consistent. 
Can you check the overall MOTA or mMOTA?

**Reply by ningqing123** [reply]
Yes，Now my mMOTA is 35% .
